examples/keymanager/keymanager_secret/main-cert-with-expiration.tf

```diff
@@ -0,0 +1,13 @@
+resource "vkcs_keymanager_secret" "certificate" {
+  name                 = "certificate-tf-example"
+  payload              = file("certificate.pem")
+  secret_type          = "certificate"
+  payload_content_type = "text/plain"
+  expiration           = timeadd(timestamp(), format("%dh", 8760)) # one year in hours
```

Comment:
we really need this?

Reply:
porbably no, as long as we use just self created cert

Reply:
secret_type is optional field but i left it because in terraform_cases this field is used